src/components/tuTable/tuTableStore.ts

```diff
@@ -1,96 +1,112 @@
+
 import _ from "lodash";
-import { Component, computed, ComputedRef, reactive, ref, Ref, shallowReactive } from "vue";
+import {
+	Component,
+	computed,
+	ComputedRef,
+	reactive,
+	ref,
+	Ref,
+	shallowReactive
+} from "vue";
 
 export class TableIdentifierAuto {
 	public static id = 0;
 }
 
 export interface TuTableRow {
-	index: number,
-	selected: boolean,
-	expanded: boolean,
-	componentValues?: {
-		[fieldName: string] : any
-	},
-	rowData?: {
-		[name:string] : any
-	}
+    index: number;
```

Comment:
I can see a lot of indentaion changes from tabs to spaces. Can we recheck if eslint is being used correctly. Essentially, all these shouldn't come in diffs